test/e2e/README.md

```diff
@@ -36,16 +40,41 @@ the object-store-provider to be specified.
 1. `-object-store-provider`: Object store provider to use. Required when kind is the cloud provider.
 1. `-velerocli`: Path to the velero application to use. Optional, by default uses `velero` in the `$PATH`
 1. `-velero-image`: Image for the velero server to be tested. Optional, by default uses `velero/velero:main`
+1. `-restore-helper-image`: Image for the velero restore helper to be tested. Optional, by default it is the built-in image address of velero image.
+1. `-plugins `: Provider plugins to be tested.
```

Comment:
I would clarify by giving examples if this should be `aws` or `velero.io/aws` or if it's a path to a plugin repo dir or if it's a container URI

Reply:
Right, actually `-plugins` should be call `-plugin-images` along with `-velero-image` and `-restore-helper-image`.

Reply:
Tho might be easier long term to just simplify this readme section to run `make test-e2e-help` or something that.. making a pr against this here.
https://github.com/danfengliu/velero/pull/53

Reply:
that way you don't have to keep updating readme.md, single place to define it.

Reply:
Adding `-help` is great, but makefile variables description (mapping to ginkgo variables) still be needed in readme file, it seems updating readme.md is not avoidable currently.